spec/clinic.v1.yaml

```diff
@@ -4164,6 +4183,31 @@ components:
           minLength: 1
         tags:
           $ref: '#/components/schemas/PatientTagIds'
+    AddServiceAccount:
```

Comment:
I don't know how much we care, but is required a better flag here than minLength 1? I guess it doesn't matter as we're not specifying any details at all really anyway.

Reply:
It is required at the moment and must not be empty.

Reply:
Are you telling me that something that's required can be empty? Seems weird to me. But yeah, I just double checked the docs, and they make no mention of it not being possible to send an empty string when a property is marked required. Seems weird, but there you go.